### PR TITLE
Add warnbanid

### PR DESCRIPTION
**Pull request by wrefgtzweve**
Add warnbanid, functions like warnban but instead uses banid

### CODE REVIEW

File: lua/ulx/modules/sh/cfc_warn_after.lua [hunk 1]
```diff
@@ -25,3 +25,16 @@ kick:addParam{ type = ULib.cmds.PlayerArg }
 kick:addParam{ type = ULib.cmds.StringArg, hint = "reason", ULib.cmds.optional, ULib.cmds.takeRestOfLine, completes = ulx.common_kick_reasons }
 kick:defaultAccess( ULib.ACCESS_ADMIN )
 kick:help( "Kicks target and warns them." )
+
+function cmd.warnBanId( calling_ply, steamid, minutes, reason )
+    ulx.banid( calling_ply, steamid, minutes, reason )
+    local command = string.format( 'awarn_warn \"%s" "%s (%s ban)" ', steamid, reason, ULib.secondsToStringTime( minutes * 60 ) )
```
Comment (brandonsturgeon) [inline lua/ulx/modules/sh/cfc_warn_after.lua:31]: I like to keep doubles outsides, singles inside. This should work

```suggestion
    local command = string.format( "awarn_warn '%s' '%s (%s ban)' ", steamid, reason, ULib.secondsToStringTime( minutes * 60 ) )
```

**Review by brandonsturgeon — CHANGES_REQUESTED** [review]
Semantic change, otherwise look good

File: lua/ulx/modules/sh/cfc_warn_after.lua (hunk 1 above)
Comment (wrefgtzweve) [reply lua/ulx/modules/sh/cfc_warn_after.lua]: The other warns had the same style, should i just change those too?

Comment (brandonsturgeon) [reply lua/ulx/modules/sh/cfc_warn_after.lua]: Eh, nah. Leave it if it's standard for the project

**Review by brandonsturgeon — APPROVED** [review]

**Pull request merged**